src/lib/blobs/blobs.ts

```diff
@@ -28,7 +28,7 @@ const printLocalBlobsNotice = () => {
 }
 
 const startBlobsServer = async (debug: boolean, projectRoot: string, token: string) => {
-  const directory = path.resolve(projectRoot, getPathInProject(['blobs-serves']))
+  const directory = path.resolve(projectRoot, getPathInProject(['blobs-serve']))
```

Comment:
Unrelated to the fix, but `blobs-serves` was a typo. It should be `blobs-serve`, in line with `functions-serve` and `edge-functions-serve`.

Reply:
I forgot to mention this! I saw the directory and guessed it was a typo.